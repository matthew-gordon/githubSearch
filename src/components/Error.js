import React from 'react'
import { StyleSheet, View, Text } from 'react-native'

export default function Error({ error }) {
  return (
    <View testId="error" style={styles.errorContainer}>
      <Text>Oh no! Something went wrong.</Text>
    </View>
  )
}

const styles = StyleSheet.create({
  errorContainer: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
})
