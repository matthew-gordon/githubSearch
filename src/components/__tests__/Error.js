import React from 'react'
import { render } from '@testing-library/react-native'

import Error from '../Error'

describe('Error', () => {
  test('renders without crashing', async () => {
    const { baseElement } = render(<Error />)

    expect(baseElement).toMatchSnapshot()
  })
})