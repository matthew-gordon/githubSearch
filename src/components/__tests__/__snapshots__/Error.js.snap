// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders without crashing 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
      testId="error"
    >
      <Text>
        Oh no! Something went wrong.
      </Text>
    </View>
  </View>
</View>
`;
